server/helpers/getPatrons.js

```diff
@@ -0,0 +1,13 @@
+const getPatrons = (data) => {
+  const temporary = [];
```

Comment:
this shouldnt be getPatrons.  the name should be more descriptive on what the function does